UIKitTemplate/UIKitTemplate/FeedScreen/FeedView.swift

```diff
@@ -0,0 +1,24 @@
+// FeedView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран ленты
```

Comment:
```suggestion
/// Вью ленты
```
экран это контроллер